pep/app/pods/glossary-term/model.ts

```diff
@@ -0,0 +1,106 @@
+import DS from 'ember-data';
+import attr from 'ember-data/attr';
+import { belongsTo } from 'ember-data/relationships';
+
+import { isEmpty } from '@ember/utils';
+
+import { INVALID_ABSTRACT_TAGS, INVALID_ABSTRACT_PREVIEW_TAGS, HTML_BODY_REGEX } from 'pep/constants/regex';
+import SimilarityMatch from 'pep/pods/similarity-match/model';
+
+export default class GlossaryTerm extends DS.Model {
```

Comment:
unless its diverged too greatly to make it worth it, this should probably just extend the `Document` model, as it looks almost identical.

Reply:
Yea I think there are fields that are not in documents. I think eventually we should create a base model to inherit from but If you don't mind we can save that for a later date

Reply:
can we just extend from `Document` for now and then just add whatever fields u are using that are not in the Document model?